examples/data-objects/multiview/triangle-view/package.json

```diff
@@ -34,9 +42,6 @@
 		"start:spo": "webpack serve --config webpack.config.cjs --env mode=spo",
 		"start:spo-df": "webpack serve --config webpack.config.cjs --env mode=spo-df",
 		"start:tinylicious": "webpack serve --config webpack.config.cjs --env mode=tinylicious",
-		"test": "npm run test:jest",
-		"test:jest": "jest",
-		"test:jest:verbose": "cross-env FLUID_TEST_VERBOSE=1 jest",
```

Comment:
Intentional?

Reply:
Yeah.  (I see you figured it below.  :-)